starky/src/lookup.rs

```diff
@@ -419,6 +427,7 @@ pub struct Lookup<F: Field> {
 }
 
 impl<F: Field> Lookup<F> {
+    /// Outputs the number of helper columns needed by this `Lookup`.
```

Comment:
Same question as above for `Lookup`. (There are a couple of others below as well, such as `GrandProductChallengeSet`)